src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/UISettings.cs

```diff
@@ -0,0 +1,187 @@
+                        return true;
+                    }
+                    catch (COMException)
+                    {
+                        // We don't want to throw any exceptions here.
+                        // If we can't get the instance, we will use the default accent color.
+                    }
+                }
+                color = _fallbackAccentColor;
+                return false;
+            }
+
+            /// <summary>
+            ///   Tries to update the accent colors properties.
+            ///   If any call to TryGetColorValue fails, we set _useFallbackColor to true.
+            ///   After which all the accent values will be the default color.
+            ///   This is to ensure that we don't have inconsistent set of accent color values stored.
+            /// </summary>
+            internal void TryUpdateAccentColors()
+            {
+                _useFallbackColor = true;
+
+                if(_isSupported)
+                {
+                    try
+                    {
+                        if(TryGetColorValue(UISettingsRCW.UIColorType.Accent, out Color systemAccent))
+                        {
+                            // For verifying if any of the calls to TryGetColorValue fails.
+                            bool result = true;
+                            if(_accentColor != systemAccent)
+                            {
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentLight1, out _accentLight1);
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentLight2, out _accentLight2);
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentLight3, out _accentLight3);
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentDark1, out _accentDark1);
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentDark2, out _accentDark2);
+                                result &= TryGetColorValue(UISettingsRCW.UIColorType.AccentDark3, out _accentDark3);
+                                _accentColor = systemAccent;
+                            }
+                            // If result is false, hence atleast one call, use fallback values.
+                            _useFallbackColor = !result;
+                        }
+                    }
+                    catch
+                    {
+                        // We don't want to throw any exceptions here.
+                        // If we can't get any one of the color values, we will use the default accent color.
+                    }
+                }
+            }
+
+            /// <summary>
+            ///   Gets the WinRT instance of UISettings.
+            /// </summary>
+            private static object GetWinRTInstance()
+            {
+                object _winRtInstance = null;
```

Comment:
I am letting pass that instance and static members have the same naming pattern, but local values should certainly be distinguishable. Don't use underscore here.